pkgs/development/web/nodejs/nodejs.nix

```diff
@@ -149,6 +151,17 @@ let
       "CI_SKIP_TESTS=test-setproctitle,test-tls-cli-max-version-1.3,test-tls-client-auth,test-child-process-exec-env,test-fs-write-stream-eagain,test-tls-sni-option,test-https-foafssl,test-child-process-uid-gid,test-process-euid-egid,test-process-initgroups,test-process-uid-gid,test-process-setgroups"
     ];
 
+    postPatch = lib.optionalString isCrossBitness ''
+      sed -i "s%@MAYBE_WRAPPER@%'<(PRODUCT_DIR)/v8-qemu-wrapper',%g" node.gyp
+      sed -i "s%@MAYBE_WRAPPER@%'<(PRODUCT_DIR)/v8-qemu-wrapper',%g" tools/v8_gypfiles/v8.gyp
+      mkdir -p out/Release
+      cat > out/Release/v8-qemu-wrapper << "EOF"
```

Comment:
Hm, I think a symlink should be enough in this case. IIRC emulator contains arguments in addition to path iff `canExecute`.
https://github.com/NixOS/nixpkgs/blob/5d49679d00dedf21e4db29218c6ac27edad7855c/lib/systems/default.nix#L285-L295

Reply:
Ah, OK.
```
nix-repl> (lib.systems.elaborate "x86_64-linux").canExecute (lib.systems.elaborate "i686-linux")   
true
```

Reply:
Yeah for i686 it needs to be a script.

Reply:
FWIW I’ve opened PR #324071 to make an emulator a simple executable path in native case.

Reply:
With that it should be possible to use only just symlink 👍

Reply:
Can we proceed with this PR and update it just once #324071 is merged. It should be later only a minor change in `postPath`. What do you think?